core_impl/src/test/java/io/opencensus/stats/ViewManagerImplTest.java

```diff
@@ -375,13 +375,13 @@ public void testViewDataWithMultipleTagKeys() {
 
   @Test
   public void testMultipleViewSameMeasure() {
-    View view1 =
+    final View view1 =
```

Comment:
I made the views final to fix this warning:

```
core_impl/src/test/java/io/opencensus/stats/ViewManagerImplTest.java:384: Distance between variable 'view2' declaration and its first usage is 4, but allowed 3.  Consider to make that variable as final if you still need to store its value in advance (before method calls that might do side effect on original value). [VariableDeclarationUsageDistance]
```

I don't know how useful this warning is, though.  Maybe we should suppress it or increase the allowed distance.